quickstarts/mlops/truera/config.yml

```diff
@@ -0,0 +1,50 @@
+# Name of the quickstart (required)
+name: truera
+
+# Displayed in the UI
+title: Truera
+
+# Long-form description of the quickstart (required)
+description: |
+  TruEra provides best in class AI Quality diagnostics and monitoring solution. With this integration, TruEra and NewRelic users can go beyond standard accuracy, input and output drift monitoring by identifying consequential data drift, and other AI Quality diagnostics such as fairness, conceptual soundness and segment behavior. When combined with NewRelic’s comprehensive observability capabilities, MLOps teams can correlate signals throughout the ML lifecycle.
```

Comment:
This is a pretty long run on sentence, would you mind trying to break this up into shorter sentences?

Reply:
You can try:
`Use the TruEra integration with New Relic and go beyond standard accuracy, input and output drift monitoring and identify consequential data drift and other AI quality diagnostics.`